cmd/proxy/main.go

```diff
@@ -163,6 +163,11 @@ func main() {
 		}(name, server)
 	}
 
+	probeStopper := readinessProbe.StartProbing()
```

Comment:
We can replace the proxy's probe with a simple TCP check on the /healthz handler